server/chat-plugins/room-events.ts

```diff
@@ -222,6 +221,25 @@ export const commands: ChatCommands = {
 			this.modlog('ROOMEVENT', null, resultString);
 			return this.sendReply(resultString);
 		},
+		enable(target, room, user, cmd) {
+			if (!this.can('gamemanagement')) return false;
+			const ranks = ['%', '@', '#'];
+			const rank = target as GroupSymbol;
+			if (!ranks.includes(target)) {
+				return this.errorReply(`Invalid rank for this setting. Valid options are [%|@|#].`);
+			}
+			if (!room.chatRoomData || room.title.includes('-')) {
```

Comment:
The `title` check doesn't really achieve anything useful, if anything you'd want a `roomid` check but even that's unnecessary when you have the `chatRoomData` check.